### ISSUE TITLE
Can't use process.stdin on windows platform

### ISSUE THREAD

**Issue by organy110** [electron/electron#21705]
### Preflight Checklist
* [ ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ ] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  7.1.8
* **Operating System:**
  Windows 10 (1903)
### Expected Behavior
process.stdin API should not be available on windows 10 if it does not work. In case electron does not support stdin on windows OS (which is very disappointing) you should throw an exception when the stdin is used and mention it **in the docs**!!

### Actual Behavior
process.stdin provides a mock stream that does nothing but sending an EOF.  lib/common/init.ts is the one responsible for this behaviour as you can see here: 
![Screenshot_20200108-203307_FastHub](https://user-images.githubusercontent.com/54235496/72073024-c6181780-32f7-11ea-8a88-0da9460c7265.jpg)

### To Reproduce
I see only 2 options to improve it:
1) Best solution would be to make process.stdin work on windows, this is not that far fetched is this api is available on many frameworks such as swing (on Java) and WPF.
2) Other solution would be to throw an exception when the stdin is used on windows (very easy to implement).

### Additional Information

I would expect this piece of code to work or at least throw an error. 

 ```typescript
        process.stdin.on("readable", data => { 
            // Gets here only once and prints undefined
            console.log(`Received data ${data}`);
        });
```

**Comment by codebytere** [comment]
We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

We will be able to more closely look at your issue once you do so. This issue will be closed in 10 days if the above is not addressed.

**Comment by electron-triage** [comment]
The Electron version reported on this issue is no longer supported. See our [supported versions documentation](https://www.electronjs.org/docs/tutorial/support#supported-versions).

**If you're still experiencing this issue on a supported version, please open a new issue with an updated repro - a [Fiddle](https://www.electronjs.org/fiddle) is very appreciated.**

Electron has a large issues backlog. To help our team prioritize, we're closing older issues and asking for new issues with updated repro steps if it affects a supported version. This helps sort what issues are still relevant and helps us fix them more quickly.

Thanks for your patience and understanding!
